src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -453,6 +460,10 @@ def set_policy(cmd, client, resource_group_name, vault_name,
     object_id = _object_id_args_helper(cmd.cli_ctx, object_id, spn, upn)
     vault = client.get(resource_group_name=resource_group_name,
                        vault_name=vault_name)
+
+    if vault.properties.enable_rbac_authorization:
+        raise CLIError('Cannot set policies to a vault with `--enable-rbac-authorization` specified')
```

Comment:
In error message we usually use single quote `'`. Grave accent `` ` `` is only for markdown.

Reply:
> In error message we usually use single quote `'`. Grave accent `` ` `` is only for markdown.

Good point, will refine this.